src/g_game.cpp

```diff
@@ -1478,7 +1478,7 @@ double FLevelLocals::PlayersRangeFromSpot (FPlayerStart *spot)
 			closest = distance_squared;
 	}
 	//and only now, when closest player are found, calculate square root
-	return sqrt(closest);
+	return std::sqrt(closest);
```

Comment:
This is literally the exact opposite of what was requested. This has to use `g_sqrt`!

Reply:
I pushed last commit before reading this pull conversation.